thanosql/parse.py

```diff
@@ -9,12 +10,15 @@ def convert_local_ns(query_string, local_ns) -> str:
     }
 
     # variables need to be converted
-    var_list = list(set(map(str.strip, re.findall(r"=( *?\w+)", query_string))))
+    var_list = list(set(map(str.strip, re.findall(r"FROM( *?\w+)", query_string))))
 
     # modifying query_string
     for i in range(len(var_list)):
         var = local_ns.get(var_list[i])
-        if var:
+        if var is not None:
```

Comment:
```suggestion
for var in var_list:
  local_var = local_ns[var] # 있는 그대로 받아오기 위해
  if isinstance(local_var, pd.DataFrame):
    local_var = local_var.to_json(orient="records", force_ascii=False)
  
  query_string = query_string.replace(var, f"'{str(local_var)}'")
return query_string
```

local_ns에 담긴거 그대로 전달하는게 맞지 않나 싶어서 한 번 바꿔 보았습니다.
검토해주세요.

Reply:
local_ns에 var이 없을 경우도 있어서 .get()으로 수정했습니다. 확인 부탁드립니다.